src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,14 @@
-export const UserInfo = () => {};
+import { User } from '../../types';
+import './UserInfo.css';
+
+type Props = {
+  user: User | null;
+};
```

Comment:
```suggestion
type Props = {
  user: User;
};
```
You don't need to render info if no user is present. And get rid of about unnecessary checks, just `user.email`